README.md

```diff
@@ -170,15 +129,15 @@ POSTGRES_DB=doccano
 
 #### Production
 
-After running the following command, access <http://0.0.0.0/>.
+After running the following command, access <http://127.0.0.1/>.
 
 ```bash
 docker-compose -f docker-compose.prod.yml --env-file ./config/.env.example up
 ```
 
 #### Development
 
-After running the following command, access <http://127.0.0.1:3000/>. If you want to use the admin site, please access <http://127.0.0.1:8000/admin/>.
+After running the following command, access <http://127.0.0.1:8000/>. If you want to use the admin site, please access <http://127.0.0.1:8000/admin/>.
```

Comment:
We need to use port `3000` to access the frontend.

Reply:
Done.